### PR TITLE
Use super create.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/projects/ProjectResourceManager.java [hunk 1]
```diff
@@ -135,22 +135,28 @@ protected Object createInternal(String type, ApiRequest request) {
...
-        List<Map<String, String>> members = (ArrayList<Map<String, String>>) project.get("members");
-        projectMemberResourceManager.setMembers(newProject, members);
-        policy.grantObjectAccess(newProject);
-
-        return newProject;
+        Object object = super.createInternal(type, apiRequest);
+        if (object instanceof Account) {
+            Account newProject = (Account) object;
+            newProject.setKind(AccountConstants.PROJECT_KIND);
+            objectManager.persist(newProject);
```
Comment (alena1108) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/projects/ProjectResourceManager.java]: Is there a reason why we don't call super.createInternal (AbstractObjectResourceManager.createInternal) here? Should be similar how we call super.updateInternal in update method.

Comment (hibooboo2) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/projects/ProjectResourceManager.java]: I do call `createinternal` On line 148.

Comment (alena1108) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/projects/ProjectResourceManager.java]: sorry, missed that. thought line 152 does the object create, but it just updates the kind

**Comment by alena1108** [comment]
LGTM

**Pull request merged**